### PR TITLE
[feature-1] First feature was implemented

### PR DESCRIPTION
**Pull request by AravindWalker**
Following changes are made :

-newfeature.txt was created.
-feature-1 folder was created.

### CODE REVIEW

File: feature-1/one-more-feature.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+One more branch in feature-1.txt
```
Comment (Aravind-byte) [inline feature-1/one-more-feature.txt:1]: It is good

Comment (Aravind-byte) [reply feature-1/one-more-feature.txt:1]: ok

Comment (Aravind-byte) [reply feature-1/one-more-feature.txt:1]: Thanks

Comment (AravindWalker) [reply feature-1/one-more-feature.txt:1]: Thanks

**Review by Aravind-byte — APPROVED** [review]
Good job

**Pull request merged**